src/IssueViz.UnitTests/IssueVisualizationControl/IssueVisualizationViewModelTests.cs

```diff
@@ -101,26 +103,109 @@ public void Ctor_InitializeWithExistingSelection()
         #region CurrentIssue PropertyChanged
 
         [TestMethod]
-        [DataRow(nameof(IAnalysisIssueVisualization.CurrentFilePath), nameof(IIssueVisualizationViewModel.FileName))]
-        [DataRow(nameof(IAnalysisIssueVisualization.Span), nameof(IIssueVisualizationViewModel.LineNumber))]
-        [DataRow(nameof(IAnalysisIssueVisualization.RuleId), null)]
-        public void CurrentIssuePropertyChanged_NotifyPropertyChanged(string issueChangedProperty, string expectedNotifiedProperty)
```

Comment:
I split this test to 3 different ones. In hindsight, I don't think it was readable, and it stopped working now that `Span` raises 2 property changes.